packages/turbo-repository/js/README.md

```diff
@@ -0,0 +1,10 @@
+## Repositories
+
+Provides JS repository analysis functionality, scoped to the JS ecosystem.
+
+Note that this is not yet a stable package, and functionality, API, and naming may all change.
```

Comment:
```suggestion
```

The version already says this, don't want to remember keep this note in sync with the version

Reply:
I think we'll need to do a significant revision of the README anyway when we commit to a more stable API. I'd like to keep this as a more explicit indication than just the version.